test/kernel_acl.js

```diff
@@ -65,14 +65,21 @@ contract('Kernel ACL', accounts => {
     it('actions cannot be performed if uninitialized', async () => {
         const newKernelProxy = await KernelProxy.new(await factory.baseKernel())
         const newKernel = Kernel.at(newKernelProxy.address)
-        return assertRevert(async () => {
-          const result = await newKernel.hasPermission(permissionsRoot, app, role, '0x00')
-        })
+        try {
+            let result = await newKernel.hasPermission(permissionsRoot, app, role, '0x00')
+            if (result.receipt) {
+                assert.equal(result.receipt.status, 0, 'should have failed status')
+            } else {
+                assert.isFalse(result, 'should not have permission')
```

Comment:
This also looks nuts—`geth` or `parity` return a result here rather than reverting?

If so, it might also be nice to have a generic wrapper for this (I notice there's another instance of this later), using a mock revert error:

```js
const assertRevertOrFalse = (code, falseMsg) => {
  return assertRevert(async () => {
    const result = await code()
    if (!result) {
      throw new Error('mock revert')
    }
  })
}
```